src/Sylius/Bundle/AdminBundle/Resources/views/Crud/_breadcrumb.html.twig

```diff
@@ -0,0 +1,23 @@
+{% import '@SyliusAdmin/Macro/breadcrumb.html.twig' as breadcrumb %}
+
+{% if breadcrumbs|default([])|length > 0 %}
```

Comment:
I'm not convinced, that this is the best condition, to append index if there will be anything in breadcrumbs. Won't it make some problems in the long term? Is there any possibility to simplify these if statements?

Reply:
I agree. 

Better solution will be like this:

```twig
{# .../Macro/breadcrumb.html.twig #}
{% macro crumble(crumbs = {}) %}
    <div class="breadcrumbs-wrapper">
        <div class="container-fluid">
            <div class="breadcrumbs">
                <ul itemscope="" itemtype="http://schema.org/BreadcrumbList">
                    {% for crumb in crumbs %}
                        <li itemprop="itemListElement" itemscope="" itemtype="http://schema.org/ListItem">
                            {% if loop.last or crumb.url|default(null) is null %} {# if url is null - this should be rendered as label rather than link #}
                                <span itemprop="item">
                                    <span itemprop="name">{{ crumb.label }}</span>
                                </span>
                            {% else %}
                                <a href="{{ crumb.url }}" title="{{ crumb.label }}" itemprop="item">
                                    <span itemprop="name">{{ crumb.label }}</span>
                                </a>
                            {% endif %}
                            <meta itemprop="position" content="{{ loop.index }}">
                        </li>
                    {% endfor %}
                </ul>
            </div>
        </div>
    </div>
{% endmacro %}
```

With that little change, at macro, _breadcrumb templates now looks much cleaner:

```twig
{# .../shop/Account/_breadcrumb.html.twig #}
{% include 'shop/_breadcrumb.html.twig' with {
    'breadcrumbs': [
        { label: 'sylius.ui.my_account'|trans, url: breadcrumbs|default([])|length > 0 ? path('sylius_shop_account_dashboard') : null }
    ]|merge(breadcrumbs|default([]))
} %}
```

Reply:
Seems nicer. Is it somehow compatible with #11607?

Reply:
Yes, current PR and 11607 is not conflicting as mine is about admin, when 11607 is about shop

Reply:
I've added breadcrumbs example from shop in this thread, but just to show how to simplify if condition.